packages/web/setupTests.js

```diff
@@ -1,2 +1,3 @@
 import '@testing-library/jest-dom/extend-expect';
 import 'jest-styled-components';
+import 'core-js';
```

Comment:
https://github.com/react-hook-form/react-hook-form/issues/1560

Reply:
Thanks a LOT for this, it really helped me out  to figure out the `getPath(...).flat is not a function` issue :)